test/fixtures/write-comparisons.js

```diff
@@ -103,5 +103,23 @@ export default {
       new Uri('remote/write-remote-script.js'),
       '<div id="local">Local</div>'
     ]
+  },
+  'nesting scripts': {
+    // Broken in IE10 see: https://github.com/krux/postscribe/issues/154
+    'skip:script written within another should complete': "<script>document.write('<script type=\"text/javascript\" src=\"remote/write-using-query-string.js?k=1&k2=2\"><\\/script>')<\/script>"
+  },
+  simplewrites: {
```

Comment:
These might as well be in here. I figured out that I was only passing in the first arg in the tests, which was broken.